src/utils/image_server.cljs

```diff
@@ -265,6 +265,10 @@
                             :ring-width               ring-width})))
 
 (defn get-account-qr-image-uri
```

Comment:
This function has no usages, why do you want to keep it in the code?

Reply:
@ilmotta 

It has an implementation in Status-go, so I'm not sure about just removing this entirely, Do you think it must be removed?